extensions/all-extensions/pom.xml

```diff
@@ -20,14 +20,19 @@
     <dependencies>
         <dependency>
             <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-file</artifactId>
+            <artifactId>mule-module-ftp</artifactId>
             <version>${project.version}</version>
             <type>zip</type>
         </dependency>
 
         <dependency>
             <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-ftp</artifactId>
+            <artifactId>mule-module-file</artifactId>
```

Comment:
mule-module-file was defined as zip type due to it has to go as extension into the distro. Is there any reason why you added only as jar dependency?